public/css/style.css

```diff
@@ -169,21 +184,179 @@ background: linear-gradient(rgba(96, 99, 100, 0.3), rgba(0,0,0,0.3));
   .play-foot, .login-btn {
     font: Bold 2rem 'Quicksand';
   }
-
-  #donations {
-    text-align: left;
+}
+.login-image-overlay {
+  width: 100%;
+  background:  linear-gradient(rgba(10, 10, 10, 0.6), rgba(10, 10, 10, 0.6)),
+      url('https://res.cloudinary.com/dqsmurjpg/image/upload/v1530770621/3-cards-against-humanity_32938.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.signin-image-overlay {
+  width: 100%;
+  background:  linear-gradient(rgba(10, 10, 10, 0.6), rgba(10, 10, 10, 0.6)),
+      url('https://res.cloudinary.com/dqsmurjpg/image/upload/v1530797290/cards_2.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+@media screen and (min-width: 767px) {
+  #cfh-logo {
+    width: 25%
   }
+}
 
-  .donation-banner {
-    padding: 3rem;
+@media screen and (max-width: 767px) {
+  #cfh-logo {
+    width: 22%
   }
+}
 
-  .wish-logo {
-    top: 28rem;
+#auth {
+  height: 83vh;
+  text-align: center;
+}
+
+@media screen and (max-width: 600px) {
+  #auth {
+    height: 85vh;
   }
+}
 
-  .nav-wrapper {
-    margin-left: 2.5rem;
-    margin-right: 2.5rem;
+.auth-input {
+  width: 400px;
+  margin-left: auto;
+  margin-right: auto;
+}
+@media screen and (max-width: 420px) {
+  .auth-input {
+    width: 300px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+@media screen and (min-width: 420px) {
+  .auth-input {
+    width: 400px;
+    margin-left: auto;
+    margin-right: auto;
   }
 }
+.input-field {
+  color:white;
+  box-sizing: border-box;
+  border: 2px solid #f1f1f1;
+  background-color: white;
+  border-radius: 3px;
+  margin-bottom: 0 !important;
+  padding: 0 10px !important;
+}
+input {
+  border-bottom: none !important;
+  height: 3.2rem !important;
+  margin: 0 !important;
+}
+.input-field input[type=text]:focus {
+  border-bottom: none !important;
+}
+.row .input-field input:focus {
+  border-bottom: none !important;
+  box-shadow: none !important
+}
+.auth-text {
+  margin-bottom: 40px !important;
+}
+.auth-btn {
+  margin-top: 30px !important;
+  height: 50px !important;
+}
+.auth-icon {
+  display: inline-block;
+  padding: 10px;
+  border: 2px solid white;
+  border-radius: 50%;
+  width: 50px;
+  height: 50px;
+}
+i {
+  color: white;
+  font-size: 24px;
+}
+.auth-component {
+  padding-top: 30px !important;
+}
+.or{
```

Comment:
Use a better class name, an example name can be `separator`. The name `or` would not make sense if you try to apply it to another element

Reply:
Done